### PR TITLE
Use Host_IsSinglePlayerGame whenever possible

### PR DESCRIPTION
**Pull request by In-line**
- [x] Find all singleplayer/multiplayer checks
- [x] Make PR build
- [x] Did we include all in this PR?
- [ ] Is it safe to use Host_IsSinglePlayerGame?

### CODE REVIEW

File: rehlds/engine/host.cpp [hunk 1]
```diff
@@ -1019,9 +1023,17 @@ void CheckGore(void)
...
+		return g_psvs.maxclients <= 1;
+#else
 		return g_psvs.maxclients == 1;
+#endif
 	else
+#ifdef REHLDS_FIXES
+		return g_pcl.maxclients <= 1;
+#else
 		return g_pcl.maxclients == 1;
+#endif
```
Comment (In-line) [inline rehlds/engine/host.cpp]: @LevShisterov `else` part only?

Comment (LevShisterov) [inline rehlds/engine/host.cpp]: All changes in this func.

Comment (In-line) [inline rehlds/engine/host.cpp]: @LevShisterov Fixed.

**Comment by In-line** [comment]
@LevShisterov Do you think it's fine in other cases ?

**Pull request closed without merging**